Homework2/Task01/task01.py

```diff
@@ -0,0 +1,65 @@
+"""
+Given a file containing text. Complete using only default collections:
+    1) Find 10 longest words consisting from largest amount of unique symbols
+    2) Find rarest symbol for document
+    3) Count every punctuation char
+    4) Count every non ascii char
+    5) Find most common non ascii char for document
+"""
+import re
+import string
+from typing import List
+
+
+def open_file(file_path: str) -> str:
+    f = open(file_path)
```

Comment:
What is recommended way to read a file and why?

Reply:
I guess you are talking about the fact that I don't close file.  We should do it to free program resources. Also we can lose some data if it's still in buffers and we haven't written it yet.